doc/source/start.rst

```diff
@@ -40,11 +40,6 @@ context and only reporting on the high-severity issues::
 
     bandit examples/*.py -n 3 -lll
 
-Bandit can be run with profiles. To run Bandit against the examples directory
```

Comment:
Bandit does still have an option to specify profiles to load via the -p or --profile argument.

Reply:
I cloned `PyCQA/bandit:main`, ran `python setup.py install` and `pip -r requirements.txt`, then ran `bandit examples/*.py -p ShellInjection`. The output is below:

```
[main]  ERROR   Unable to find profile (ShellInjection) in config file: None
```

This result suggests that profile is not built-in, but something we define ourselves. So how do we define it?
Where is it written in the documentation?

Reply:
Yes, creating profiles is still possible via the bandit.yaml file.  They are not built-in. And you can create a config yaml file bandit-config-generator

Reply:
Thank you. I ran `bandit-config-generator -o bandit.yaml` then `bandit examples/*.py -p ShellInjection -c bandit.yaml`. The output is:

```
[main]  ERROR   Unable to find profile (ShellInjection) in config file: bandit.yaml
```

Again, how can I define profile in a configuration file?

After some trial and error, a configuration file as below finally worked.

```yaml
profiles:
  ShellInjection:
    # ...
```

But I got the message below.

```
[config]        WARNING Config file 'bandit.yaml' contains deprecated legacy config data. Please consider upgrading to the new config format. The tool 'bandit-config-generator' can help you with this. Support for legacy configs will be removed in a future bandit version.
```

This is probably the best evidence that profile is deprecated and should not be used.
Therefore, all descriptions about profile should be removed from the documentation.

Reply:
Even if it's "deprecated" (which it isn't) it shouldn't be removed in this pull request

Reply:
I see. Reverted in 2d6cd55